src/components/number-picker/number-picker.component.spec.ts

```diff
@@ -187,6 +219,30 @@ describe('Number Picker Component - ngModel', () => {
         expect(numberPicker.classList.contains('ux-number-picker-invalid')).toBe(true);
     });
 
+    it ('should not display ux-number-picker-invalid class when max value increased to match input value', async() => {
+        component.value = 15;
+        fixture.detectChanges();
```

Comment:
We should probably assert that the state is invalid at this point in this and the other related tests.